README.md

```diff
@@ -21,7 +22,7 @@ See the [CerberusClient](http://engineering.nike.com/cerberus-node-client/Cerber
 ## Authentication
 
 The cerberus client uses the [AWS SDK Credentials provider chain][node aws credentials] to load AWS IAM credentials and authenticates with Cerberus via the [sts auth endpoint](https://github.com/Nike-Inc/cerberus-management-service/blob/master/API.md#app-login-sts-v2-v2authsts-identity)
-This client will work in any environment that has access to AWS Credentials.
+This client will work in any vironment that has access to AWS Credentials.
```

Comment:
typo: `vironment` should be `environment`

Reply:
fixed